app/config.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
+    smtp_username: str = "no-username"
+    smtp_password: str = "no-password"
+    smtp_from_email: str = "invite@calendar-app.com"
+    smtp_port: int = 3999
+    smtp_server: str = "localhost"
+    smtp_use_tls: bool = False
+    smtp_use_ssl: bool = False
+    smtp_use_credentials: bool = False
+
+
+def get_settings():
+    return Settings()
```

Comment:
That's a good way of doing so, but we already have a configuration file :) Please take a look at [`config.py.example`](https://github.com/PythonFreeCourse/calendar/blob/develop/app/config.py.example).

When the user clones the repo, he should copy it and either:

1. Create his own `config.py` with his own configurations
2. Create his own `config.py` and set his environment variables according to the variable names in the file.

Reply:
oyy
I need this for my mail tests because i use other configurations for tests then not tests
this way you can put environment vairables or default values
like in here: [https://fastapi.tiangolo.com/advanced/settings/#create-the-settings-object](url) 
and then i use this: [https://fastapi.tiangolo.com/advanced/settings/#settings-and-testing](url) for tests

in the mail team we worked a lot on making this work
is it ok if I add database and media sopport to this way of configuration?
all their tests work good for me when doing this

Reply:
ok